oneandone/src/test/java/org/apache/jclouds/oneandone/rest/internal/BaseOneAndOneLiveTest.java

```diff
@@ -102,6 +105,16 @@ protected Server createServer(String serverName) {
               .powerOn(Boolean.TRUE).build());
    }
 
+   protected Server updateServerStatus(Server server) {
```

Comment:
Why do we need to use this? Is this a common pattern users will need to use when consuming this api? If this is the case, we should consider providing predicates that block until the resource is ready.